tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -122,3 +122,46 @@ Feature: spaces management
       | David | viewer  |
       | Edith | viewer  |
     And "Alice" logs out
+
+
+  Scenario: admin user can manage the spaces created by other space admin user
```

Comment:
```suggestion
  Scenario: admin user can manage all spaces in the admin settings via the context menu
```

something like this

Reply:
its the specific scenario where admin manges the spaces created by other users so i mention `created by other user`.

Reply:
i think all spaces include space created by the admin itself and another user. But i am not sure

Reply:
i think all spaces include space created by the admin itself and another user. But i am not sure